samples/nfc/tag_reader/src/main.c

```diff
@@ -26,12 +27,13 @@
 #define NFCA_FDT_ALIGN_20 20
 
 #define NFC_T2T_READ_CMD 0x30
-
 #define NFC_T2T_READ_CMD_LEN 0x02
-#define NFC_T2T_BLOCK_LEN 0x10
 
-#define NFC_TX_DATA_LEN NFC_T2T_READ_CMD_LEN
-#define NFC_RX_DATA_LEN NFC_T2T_BLOCK_LEN
+#define NFC_T4T_ISODEP_FSD 256
```

Comment:
Don't these changes somehow affect the readme.rst of the sample?

Reply:
This changes haw some impact on readme.rst but this PR contains only part of this changes. I will align the documentation in last PR with completed functionality of Type 4 Tag.